todo_item.py

```diff
@@ -0,0 +1,40 @@
+from config import KEY, TOKEN
+import requests
+from datetime import datetime
+
+class todo_item:
```

Comment:
This should be called `TodoItem` (see pep 8 for the official python styling guide)